cmd/tarmak/cmd/root.go

```diff
@@ -70,7 +71,10 @@ func stringSliceContains(s []string, e string) bool {
 
 func init() {
 	// set default tarmak config folder
-	tarmakConfigPath := "~/.tarmak"
+	tarmakConfigPath, err := homedir.Expand("~/.tarmak")
+	if err != nil {
+		fmt.Printf("unable to detect home directory: %s", err)
```

Comment:
I feel we should panic here instead of printing that there was a problem - if err is non-nil then tarmakConfigPath won't be set correctly and tarmak will fail more confusingly somewhere later

Reply:
`os.Exit(1)` would do I guess.

Reply:
I will add `os.Exit(1)` after the Printf so that we have a clear error message and when we shut it down. ok @dippynark @charlieegan3 ?

Reply:
Sounds good :+1:

Reply:
Added it